doc/reduce-memory.md

```diff
@@ -16,10 +16,14 @@ The size of some in-memory caches can be reduced. As caches trade off memory usa
   - The minimum value for `-maxmempool` is 5.
   - A lower maximum mempool size means that transactions will be evicted sooner. This will affect any uses of `bitcoind` that process unconfirmed transactions.
 
-- To completely disable mempool functionality there is the option `-blocksonly`. This will make the client opt out of receiving (and thus relaying) transactions completely, except as part of blocks.
+- To disable mempool functionality there is the option `-blocksonly`. This will make the client opt out of receiving (and thus relaying) transactions, except for from whitelisted peers and as part of blocks.
 
   - Do not use this when using the client to broadcast transactions as any transaction sent will stick out like a sore thumb, affecting privacy. When used with the wallet it should be combined with `-walletbroadcast=0` and `-spendzeroconfchange=0`. Another mechanism for broadcasting outgoing transactions (if any) should be used.
 
+  - In `-blocksonly` mode the node will by default use a 5MB mempool size, avoiding unexpected memory usage from dbcache sharing the 300MB default mempool cache size.
+
+  - If you would like to continue using the mempool cache allocation for dbcache in `-blocksonly` mode then you can specify the mempool size using the `maxmempoolsize` option.
```

Comment:
nit: Maybe also add a note saying that setting `-limitdescendantsize` can also alter the cache size with `-blocksonly`?

`./src/bitcoind -limitdescendantsize=0`
> Using 440.0 MiB for in-memory UTXO set (plus up to 286.1 MiB of unused mempool space)

`./src/bitcoind -limitdescendantsize=300`
> Using 440.0 MiB for in-memory UTXO set (plus up to 286.1 MiB of unused mempool space)

---
`./src/bitcoind -limitdescendantsize=0 -blocksonly`
> Using 440.0 MiB for in-memory UTXO set (plus up to 0.0 MiB of unused mempool space)

`./src/bitcoind -limitdescendantsize=300 -blocksonly`
> Using 440.0 MiB for in-memory UTXO set (plus up to 11.4 MiB of unused mempool space)

Reply:
Hmmm you are right. I wanted to avoid another magic constant if possible, but this does indeed have unintended side effects. I am going to revert it to 5MB for now